include/seqan3/range/view/pairwise_combine.hpp

```diff
@@ -288,8 +288,8 @@ class pairwise_combine_view : public ranges::view_interface<pairwise_combine_vie
         //!\}
 
         /*!\name Comparison operators
-         * \brief These operators are available if the `underlying_iterator_type` models std::EqualityComparable or
-         *        std::StrictTotallyOrdered respectively.
+         * \brief These operators are available if the `underlying_iterator_type` models std::equality_comparable or
+         *        std::totally_ordered respectively.
```

Comment:
The word strict is gone, is that intentional?

Reply:
yep